crypto/x509/x509_vfy.c

```diff
@@ -870,10 +886,8 @@ static int check_trust(X509_STORE_CTX *ctx, int num_untrusted)
          * we'll accept X509_TRUST_UNTRUSTED when not self-signed.
          */
         trust = X509_check_trust(mx, ctx->param->trust, 0);
-        if (trust == X509_TRUST_REJECTED) {
-            X509_free(mx);
```

Comment:
Why are we now choosing to leak "mx" (skip the X509_free()?)

Reply:
Oops, this was by mistake when handling non-trivial merge conflicts. Fixed.